source/Backend/DataBase/QueryOperationBuilder.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <format>
+#include <functional>
+#include <map>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+
+#include "common/DataBaseCommon.hpp"
+#include "common/DataBaseRequest.hpp"
+#include "interface/IQueryOperationBuilder.hpp"
+
+namespace dataBase
+{
+
+  class QueryOperationBuilder : public interface::IQueryOperationBuilder
+  {
+  public:
+    using QueryBuilders = std::unordered_map<
+        common::DataBaseRequest::OperationType,
+        std::function<std::string(const common::DataBaseRequest &,
+                                  const std::string &)>>;
+    using OperationType = common::DataBaseRequest::OperationType;
+    using TableType = common::DataBaseRequest::TableType;
```

Comment:
If this is only used in cpp file it is better to leave it there :)

Reply:
You right, I will put this in to anonymous namespace